internal/pkg/plugininstaller/jenkins/validate.go

```diff
@@ -111,18 +112,25 @@ func generateRandomSecretToken() string {
 	return fmt.Sprintf("%x", b)[:32]
 }
 
-func buildCIConfig(jenkinsFilePath string) *ci.CIConfig {
+func buildCIConfig(opts *JobOptions) *ci.CIConfig {
+	jenkinsFilePath := opts.JenkinsfilePath
 	ciConfig := &ci.CIConfig{
 		Type: "jenkins",
 	}
 	// config CIConfig
-	url, err := url.ParseRequestURI(jenkinsFilePath)
+	jenkinsfileURL, err := url.ParseRequestURI(jenkinsFilePath)
 	// if path is url, download from remote
-	if err != nil || url.Host == "" {
+	if err != nil || jenkinsfileURL.Host == "" {
 		ciConfig.LocalPath = jenkinsFilePath
 	} else {
 		ciConfig.RemoteURL = jenkinsFilePath
 	}
+	harborURLHost := opts.getHarborHost() + "/"
```

Comment:
Why not just use `path.Join` in the code to complete `repository` and pass in only one variable for the template, when using these three variables in the template?

Reply:
```go
"Repository": path.Join(opts.getHarborHost(), defaultHarborProject, opts.ProjectRepo.Repo)
```